package.json

```diff
@@ -8,6 +8,7 @@
     "@storybook/builder-vite": "^0.2.2",
     "awesome-phonenumber": "^3.2.0",
     "pretty-quick": "^3.1.3",
+    "react-flatten-children": "^1.1.2",
```

Comment:
Do you need this library?

Reply:
🤮 I would like it not to be a dependency, but it conflicts with our tsconfig and I am not patient enough to fix it.